configuration/wireless/access-point-bridged.md

```diff
@@ -120,6 +120,21 @@ interface br0
 ```
 With this line, interface `br0` will be configured in accordance with the defaults via DHCP. Save the file to complete the IP configuration of the machine.
 
+<a name="wifi-cc-rfkill"></a>
+## Ensure wireless operation
+
+Countries around the world regulate the use of telecommunication radio frequency bands to ensure interference-free operation. The Linux OS [helps](https://wireless.wiki.kernel.org/en/developers/regulatory/statement) users comply with these rules by simply configuring applications with a 2-letter "WiFi country code", e.g. `US` for a computer used in the United States.
+
+In the Raspbian OS, 5 GHz wireless networking is disabled until a WiFi country code has been configured by the user, usually as part of the initial installation process (see wireless configuration pages in this [section](./) for details.)
```

Comment:
I think we usually do relative links like `[this section](README.md) ` ?

Reply:
Sorry for that. I had trouble formulating a link to would point to the default page on the same dir. "./" worked but I wasn't too proud of it.
There is the same in the other doc.

Reply:
I finally understood I could modify my repo post-PR and solve an issue you find here... README.md links updated.